### PR TITLE
Added stock name to success response

### PR DESCRIPTION
**Pull request by shykeiichi**
Added stock name on a successfull response as it's an easy implementation and quite nice to have in a stock api.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -30,10 +30,14 @@ class YahooStockAPI {
             if ($('title').text() == 'Requested symbol wasn\'t found') throw new Error('Symbol not found!');
             let currency: string | undefined = $('#quote-header-info > div:nth-child(2) > div > div > span').text();
             currency = currency ? currency.split('.')[1].replace('Currency in', '').trim() : undefined;
+            let name: string | undefined = $('h1').text();
+            const regex = /^[^(]*/;
+            name = regex.exec(name)[0].substring(0, name.length - 2);
```
Comment (phamleduy04) [inline src/index.ts:35]: use `trim()` to remove excess spaces

File: src/index.ts [hunk 2]
```diff
@@ -52,12 +56,15 @@ class YahooStockAPI {
             const responseBody = await request.body.text();
             const $ = cheerio.load(responseBody);
             let currency: string | undefined = $('#quote-header-info > div:nth-child(2) > div > div > span').text();
+            let name: string | undefined = $('h1').text();
+            const regex = /^[^(]*/;
+            name = regex.exec(name)[0].substring(0, name.length - 2);
```
Comment (phamleduy04) [inline src/index.ts:61]: here too `trim()`

**Comment by phamleduy04** [comment]
hi, thanks for your PR. I think because both of these blocks look similar, should we create a function so the codebase doesn't get duplicated too much? Thank you

**Comment by shykeiichi** [comment]
Moved the name tidying code to a function named getTidyName in the same class

File: src/index.ts [hunk 3]
```diff
@@ -52,12 +60,13 @@ class YahooStockAPI {
             const responseBody = await request.body.text();
             const $ = cheerio.load(responseBody);
             let currency: string | undefined = $('#quote-header-info > div:nth-child(2) > div > div > span').text();
+            let name: string = this.getTidyName($('h1').text());
```
Comment (phamleduy04) [inline src/index.ts:63]: `const` instead of `let`

**Comment by phamleduy04** [comment]
LGTM! Thanks for your PR

**Comment by phamleduy04** [comment]
@all-contributors please add @twhtanghk for code

**Comment by phamleduy04** [comment]
@all-contributors please add @shykeiichi for code

**Comment by phamleduy04** [comment]
my bad lol

**Pull request merged**